Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.6.3
```

Comment:
Given that Red October is a statically linked Go binary, why not use a simpler base with no compiler in it, like a plain Debian or Alpine image, or scratch?